### PR TITLE
[OAuth][Login] Fix oauth login

### PR DESCRIPTION
**Pull request by gperdomor**
| Q               | A
| --------------- | ---
| Bug fix?        |yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT
| Doc PR          | -

This fix the oauth login/registration

### CODE REVIEW

**Comment by gperdomor** [comment]
Because this create a customer, maybe we need fire <code>sylius.customer.register</code> event. WDYT?

File: src/Sylius/Bundle/UserBundle/spec/OAuth/UserProviderSpec.php [hunk 1]
```diff
@@ -154,6 +154,7 @@ function it_should_create_new_user_when_none_was_found(
         $response->getEmail()->willReturn(null);
         $response->getUsername()->willReturn('username');
         $response->getNickname()->willReturn('user');
+        $response->getRealName()->willReturn('Name');
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/UserBundle/spec/OAuth/UserProviderSpec.php]: Missing `$customer->setFirstName('Name')->shouldBeCalled()`

Comment (gperdomor) [inline src/Sylius/Bundle/UserBundle/spec/OAuth/UserProviderSpec.php]: Done :)

**Comment by michalmarcinkowski** [comment]
Thank you @gperdomor :+1:

**Pull request merged**